.github/workflows/test_filenames_check.yml

```
@@ -0,0 +1,24 @@
+name: Test Filenames Check
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ opened, synchronize, reopened ]
+
+jobs:
+  test_filenames_check:
+    name: Test Filenames Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Filename Inspector
+        id: scan-test-files
+        uses: AbsaOSS/filename-inspector@master
+        with:
+          name_patterns: '*UnitTest.*,*IntegrationTest.*'
```

it actually must be plural; that's the reason why my UT was skipped before 

```suggestion
          name_patterns: '*UnitTests.*,*IntegrationTests.*'
```

---

also here in the README I can see that it's wrong: https://github.com/AbsaOSS/filename-inspector

or rather..if we want to be consistent then let's agree on the common practice? @miroslavpojer

---

Interesting. Wonder, how it has passed actually. 🤔 @miroslavpojer ?

---

I think you are equating a file name inspector with our practice. Does the README example have to be how we practice it in AUL?

---

README might have a wrongly described input parameters. Instead of `_` it uses `-` for input parameters.

---

> I think you are equating a file name inspector with our practice. Does the README example have to be how we practice it in ~AUL~ Atum?

The team decides on project practice and naming rules.

---

btw the CI is still failing for this reason: https://github.com/AbsaOSS/atum-service/actions/runs/9595868833/job/26461518865?pr=212

---

Now, it is failing as expected. See plural names of test files vs single ones in action config.

---

Some exclusions, i.e. false alarms presence, might occur, as they did now. Feature request? :)